src/background.js

```diff
@@ -87,3 +125,17 @@ async function enroll(isTreatmentBranch) {
     sendTestMessage("ready");
   });
 })();
+
+async function onTabUpdated(tabId, changeInfo, tabInfo) {
```

Comment:
Feel free to throw this out if it's not compatible with your approach to background.js. I just needed some live functionality so the test would run.